apps/meteor/server/models/raw/Rooms.js

```diff
@@ -466,6 +466,24 @@ export class RoomsRaw extends BaseRaw {
 		]);
 	}
 
+	findByBroadcast(options) {
+		return this.find(
+			{
+				broadcast: true,
+			},
+			options,
+		);
+	}
+
+	findByActiveLivestream(options) {
```

Comment:
what does `Active` mean here? just asking =)

Reply:
It means that the room announcement hasn't been cleared yet. When a livestream ends, all the content from the `streamingOptions` object is erased, so it's safe to use this field to check if there's an ongoing livestream.